test/test_launcher.rb

```diff
@@ -44,7 +44,7 @@ def test_state_permission_0640
 
     create_launcher(conf).write_state
 
-    assert File.stat(state_path).mode.to_s(8)[-4..-1], state_permission
+    assert_equal File.stat(state_path).mode.to_s(8)[-4..-1], state_permission
```

Comment:
This test fails :/

Reply:
Yes, saw that.  Thanks for looking for all the `assert` statements.

I'm not sure what wrong with that.  Can't look at right now...